cernopendata/modules/fixtures/data/articles/cms-guide-for-education/education.md

```diff
@@ -1,73 +1,67 @@
-
-**If you are really into this and you want to learn how data analysis is done by scientists in the CMS experiment:**
-- install the Virtual Machine (VM) as instructed in [CMS 2011 Virtual Machines: How to install](/articles/cms-2011-virtual-machines-how-to-install)
-- follow the instructions in [Getting Started with CMS Open Data](/articles/getting-started-with-cms-open-data)
-- or see an extensive overview of research activities in [Guide for research use of CMS Open Data](/articles/cms-guide-to-research-use-of-cms-open-data).
-
-**If you want to discover the Higgs boson in the CMS Open Data:**
-- install the Virtual Machine (VM) as instructed in [CMS 2010 Virtual Machines: How to install](/articles/cms-2010-virtual-machines-how-to-install) (if you did not do it already)
-- see [Higgs analysis example](TOBEFIXED).
+- If you already know what the basic physics quantities measured in a particle collision are and want to **play online with histograms** from different samples of open data, check out the [CMS histogram visualiser](http://opendata.cern.ch/visualise/histograms/CMS). For instructions on how to use the visualiser, click on `Need help?`.
+- To learn some concepts of statistical analysis in a hands-on classroom activity, have a look at the material for an organised ["**Masterclass**" exercise](/record/53) based on CMS data and the online event display. A CMS Masterclass typically takes one day (including introductory lectures and post-analysis discussions), but the exercise itself can be done in approximately two hours. Learn more about organising your own Masterclass [on the CMS website](https://cms.cern/engage-with-cms/cms-physics-masterclass).
+- You can also use **spreadsheet programs** to make a further step in the direction of data analysis, using [a limited number of CMS events that are suitable for this type of exercise](TOBEFIXED). These exercises can be done in an hour or so.
+- For a first taste of real programming using physics data, consider these resources that use [**Jupyter notebooks**](https://jupyter.org/) for a browser-based introduction to data analysis (time can vary from less than an hour to several hours, depending on the scope of your exercise):
+    - [A quick (completely online) hands-on demo](https://mybinder.org/v2/gh/cms-opendata-education/cms-online-notebooks-for-binder/master?filepath=quick-start-to-CMS-open-data.ipynb) to Jupyter and CMS Open Data (using python).
+    - [Analysing CMS Open Data in Jupyter using **python**](TOBEFIXED)
+    - [Analysing CMS Open Data in Jupyter using **R**](TOBEFIXED), with a brief intro to the R programming language
 
-Have fun!!
+---
 
+#### Dive deeper into the data
 
+If you want a more detailed understanding of particle physics or an introduction aimed at the university level, take a look at some of the other resources in [CMS Learning resources ](http://opendata.cern.ch/collection/CMS-Learning-Resources).
```

Comment:
@tiborsimko Are collection pages like _http://opendata.cern.ch/collection/CMS-Learning-Resources_ ported to _/search?q=CMS-Learning-Resources_ ?

Reply:
@tiborsimko: Please let me know if I should change this before the PR is merged. Thanks!